addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -38,8 +38,9 @@ def get_html(self, bom_id=False, searchQty=1, searchVariant=False):
         return res
 
     @api.model
-    def get_bom(self, bom_id=False, product_id=False, line_qty=False, line_id=False, level=False):
+    def get_bom(self, bom_id=False, product_id=False, line_qty=False, line_id=False, level=False, has_attachments=False):
```

Comment:
What is the purpose of this kwargs?

Reply:
We have added this params as we need to hide column ( attachement ) in report when no attachement in bom line or sub bom line.